src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java

```diff
@@ -358,6 +358,25 @@ public void scenario5() throws IOException {
     }
 
     @Test
+    public void scenarioNoAcl() throws IOException {
+        final String id = "/rest/" + getRandomUniqueId();
+        final String testObj = ingestObj(id);
+
+        logger.debug("Anonymous can't read");
```

Comment:
According to the "spec", foaf:agent would imply "public" as opposed to "any authenticated":
https://www.w3.org/wiki/WebAccessControl#Resources_covered_by_the_ACL

...which would lead me to believe that an authorization (like the root-authorization.rdf) that specifies `acl:Read` for `foaf:Agent` would indeed allow anonymous read.